GCMD command changes

Commodore64x opened the pull request:
Added GCMD blacklist and blocked from console, resolves ~~#1905 #1899 and #1900~~ Issues superseded by #1906

Commit:
GCMD command changes

Added GCMD blacklist and blocked from console, resolves #1905 #1899

Commit:
Update 

Added GCMD blacklist and blocked from console, resolves #1905 #1899

Commit:
Update Command_gcmd.java

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/TotalFreedom/TotalFreedomMod?pullRequest=1906) <br/>All committers have signed the CLA.

Commit:
Update Command_gcmd.java

lemonsked commented on src/main/java/me/totalfreedom/totalfreedommod/command/Command_gcmd.java:
```diff
@@ -1,56 +1,83 @@
+        catch (Exception e)
         {
-            msg("Error sending command: " + ex.getMessage());
+            msg("/gcmd <fromname> <outcommand>");
```
Just use return false

lemonsked replied:
Stil. needs this @Commodore64x

Commodore64x commented:
Have added in message for GCMDing players, resolves #1900

Commit:
Update Command_gcmd.java

ghost commented:
Signed the CLA.

ghost commented:
Testing for bugs now :P

ghost commented:
I think some styling could be added to the message that alerts that a command has been run by you, any suggestions?

Commit:
Command GCMD changes

Added in a message to say that you have been gcmded (Bug fixed)
Added a blacklist for certain commands (Big fixed)
Blocked GCMD from console (Bug fuxed)
Block repeat gcmd commands: e.g. Gupta issues command: /gcmd gupta2 gcmd gupta3 etc. (Bug fixed)

ghost commented:
Fixed: Sender coming up as CraftPlayer
Fixed: Command not being blocked

ghost commented:
I am pretty satisfied with this at the moment, if some other people want to test, go right ahead.

ghost commented:
Lemon please remove the Needs CLA tag.

mayokek commented:
I thought you can only create a PR per accepted issues.

ghost commented:
@AvalancheYT at this point, the things that this PR features are very unlikely to get rejected. They will definitely help with gcmd command abuse by certain admins.

Wild1145 commented:
This PR will remain on hold until the associated issue is reviewed, discussed and approved.

Please also try to keep it to one PR to one Issue, not one PR for multiple issues.

Commit:
Update Command_gcmd.java

Wild1145 reviewed (commented):
I can see no file changes... At all... You appear to have 7 commits that do nothing...

mayokek commented:
@wild1145, he have deleted the changes.

ghost commented:
What??? I haven't deleted anything... Let me check @Wild1145

ghost commented:
Superseeded by #1911
